wrappers/gin/gin.go

```diff
@@ -127,12 +127,19 @@ func (router *GinRouterWrapper) HEAD(relativePath string, handlers ...gin.Handle
 	return router.Handle(http.MethodHead, relativePath, handlers...)
 }
 
+// Run is a shortcut for router.IRouter.(*gin.Engine).Run()
+func (router *GinRouterWrapper) Run(addr ...string) error {
+	return router.IRouter.(*gin.Engine).Run(addr)
```

Comment:
The IRouter doesn't have to be of time `*gin.Engine` necessarily, so you have to add type conversion check:
``````
```suggestion
        engine, ok := router.IRouter.(*gin.Engine)
        if ok {
            return engine.Run()
        }
        return fmt.Erorrf(...)
```